specification/columns/listcost.md

```diff
@@ -26,7 +26,7 @@ Cost calculated by multiplying List Unit Price and the corresponding Pricing Qua
 | Constraint      | Value                   |
 |:----------------|:------------------------|
 | Column type     | Metric                  |
-| Column required | True                    |
+| FOCUS Essential | True                    |
```

Comment:
Does this require that the vendor publish a list cost...though that may not actually be published? I see this as a stumbling block for many vendors to adopt even single line item focus data.

Reply:
Question answered in discussion today. @ijurica  perhaps we ensure it is clear that in the absence of a list price it matches BilledCost.

Reply:
We're not specifying that it needs to be 'public' anywhere. Many providers don't publish rates publicly but they will privately tell you the starting point for price/cost (list price/cost) and then go from there to a negotiated price/cost for you.

Reply:
"The List Unit Price represents the suggested provider-published unit price for a single Pricing Unit of the
associated SKU, exclusive of any discounts."

Reply:
I interpreted "provider-published" to mean that I must publicly "publish" prices. If we remove that, the sentence still conveys the same point. Maybe we just simplify?

> _The List Unit Price represents the suggested unit price for a single Pricing Unit of the
associated SKU, exclusive of any discounts._

Reply:
> The List Unit Price represents the Provider suggested unit price for a single Pricing Unit of the
associated SKU, exclusive of any discounts.

How about this?

Reply:
Created discussion topic #366 to continue identification of a resolution for this